articles/user-guide/quickref/iqsharp.md

```diff
@@ -27,12 +27,12 @@ no-loc: ['Q#', '$$v']
 ### Azure Quantum integration
 
 - [`%azure.connect`](xref:microsoft.quantum.iqsharp.magic-ref.azure.connect): Connects to an Azure Quantum workspace or displays current connection status.
-- [`%azure.execute`](xref:microsoft.quantum.iqsharp.magic-ref.azure.execute): Executes a job in an Azure Quantum workspace.
+- [`%azure.execute`](xref:microsoft.quantum.iqsharp.magic-ref.azure.execute): Runs a job in an Azure Quantum workspace.
```

Comment:
"Submits a job to an Azure Quantum workspace and waits for completion." Similar update should also be made at https://github.com/microsoft/iqsharp/blob/740965405b813105b80e5b57d3714be9f0842f99/src/AzureClient/Magic/ExecuteMagic.cs#L32.